src/paragraph/Paragraph.style.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components'
+
+import { space } from '../_utils/branding'
+
+export const StyledParagraph = styled.div`
```

Comment:
why semantically paragraph isn't a `p`?

Reply:
I do not change the component cause it was previously like this and it'll impact lot of snapshot in Kairos.
I suggest to fix this semantic issue in an other specific task.